open-sdg/open-sdg
# Cookie consent form

Q | A
--- | ---
Feature branch/test site URL | [Link (just essential)](http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-cookies/)<br> [Link (with GA)](http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-site-cookies-analytics/)
Fixed issues | Fixes #1231 
Related version | 1.4.0-dev
Bugfix, feature or docs? | Feature
Keeps backward-compatibility? |✔️
Includes tests? |❌
Updated docs? |✔️
Updated config forms? |✔️
Added CHANGELOG entry? |❌

## Testing instructions

* This depends on translations - the "cookies" branch of https://github.com/brockfanning/sdg-translations
* To enable, use this site config:

        cookie_consent_form:
            enabled: true

* We should also test with it disabled to make sure everything still works (contrast modes, Google Analytics, and any other javascript functionality)
